### PR TITLE
Add a setting to toggle the audio proximity alarm/feedback

### PR DESCRIPTION
**Pull request by nirvn**
A new precise view menu item has been added to toggle the audio proximity alarm/feedback:
image

When unchecking the menu item, QField will keep silent. 

In addition to the permanent on/off toggle, a new mute/snooze alarm button is now present in the precise view whenever the distance to destination is within the precise view visibility threshold:
image

When hitting the snooze/mute alarm button, the audio alarm feedback will be muted *until* the distance to destination becomes higher than the precise view visibility threshold, beyond which the snooze/mute state is toggled off and the audio alarm feedback will return whenever the distance falls back within the above-mentioned threshold.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 650768c3b8d83f33bb7523339045e75b6f07f359: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3120-1daa54365995af0b820b4d64d9430c5151260185-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3120-1daa54365995af0b820b4d64d9430c5151260185-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3120-1daa54365995af0b820b4d64d9430c5151260185-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3120-1daa54365995af0b820b4d64d9430c5151260185-x86-android.apk)
</details>

**Comment by duke-nyuki** [comment]
Task linked: [QF-2069 Add "mute" functionality](https://app.clickup.com/t/2192114/QF-2069)

File: src/core/positioning/internalgnssreceiver.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ void InternalGnssReceiver::handlePositionUpdated( const QGeoPositionInfo &positi
                                                             speed,
                                                             direction,
                                                             mSatellitesInfo, 0, 0, 0,
-                                                            hacc,
+                                                            0.01, //hacc,
```
Comment (m-kuhn) [inline src/core/positioning/internalgnssreceiver.cpp:128]: What's that?

Comment (nirvn) [reply src/core/positioning/internalgnssreceiver.cpp:128]: That sir is the only way to test things out on a machine that doesnt have a gnss device ;) I'll revert.

Comment (nirvn) [inline src/core/positioning/internalgnssreceiver.cpp:128]:
```suggestion
                                                            hacc,
```

**Review by m-kuhn — APPROVED** [review]
@lucienicolier can you do the video with this?

**Comment by lucienicolier** [comment]
works well

**Comment by lucienicolier** [comment]
@m-kuhn I need it on dev to be able to use it with the tablet.

**Pull request merged**